.gitlab-ci.yml

```diff
@@ -221,7 +255,7 @@ verify:branch:
   needs:
     - build:ts
     - build:native
```

Comment:
i guess these could probably can be removed?

Reply:
Maybe. I don't think we need build:ts but we may need build:native. Build:native does the yarn rebuild. The duplicated stuff isn't persisted as an artifact. They do no harm

Reply:
we didn't change anything in the development pipeline

Reply:
That too.